options.html

```diff
@@ -3,67 +3,90 @@
 	<head>
 		<title>Video Speed Controller: Options</title>
 		<link rel="stylesheet" href="options.css" />
-		<script src="options.js"></script>
+        <script src="jquery-3.2.1.slim.min.js"></script>
+        <script src="options.js"></script>
 	</head>
 	<body>
 		<header>
 			<h1>Video Speed Controller</h1>
 		</header>
 
-    <section>
+    <section id="customs">
       <h3>Shortcuts</h3>
-      <div class="row">
-        <label for="rewindKeyInput">Rewind</label>
-        <input id="rewindKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
-      <div class="row">
-        <label for="advanceKeyInput">Advance</label>
-        <input id="advanceKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
-      <div class="row">
-        <label for="resetKeyInput">Reset speed</label>
-        <input id="resetKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
-      <div class="row">
-        <label for="slowerKeyInput">Decrease speed</label>
-        <input id="slowerKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
-      <div class="row">
-        <label for="fasterKeyInput">Increase speed</label>
-        <input id="fasterKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
-      <div class="row">
-        <label for="fastKeyInput">Preferred speed</label>
-        <input id="fastKeyInput" placeholder="press a key" type="text" value=""/>
-      </div>
       <div class="row">
         <label for="displayKeyInput">Show/hide controller</label>
         <input id="displayKeyInput" placeholder="press a key" type="text" value=""/>
       </div>
-      <div class="row">
-          <label for="startHidden">Hide controller by default</label>
-          <input id="startHidden" type="checkbox"/>
-      </div>
-    </section>
+        <div class="row customs">
+            <select class="customDo">
+                <option value="slower">Decrease speed</option>
+            </select>
+            <input class="customKey" type="text" value="" placeholder="press a key">
+            <input class="customValue" type="text" placeholder="value (0.10)">
+            <select class="customReturn">
+                <option value="0">Do not disable website key bindings</option>
+                <option value="1">Disable websites key bindings</option>
```

Comment:
^ what are these for? In practice we cannot guarantee that we can disable or override site-provided keybindings.

Reply:
it is an experimental feature which is good for pause. that way we can use space bar to pause video even in livestreams(like twitch).

Reply:
I'd propose we decouple experimental features from this PR. 

If we ship this, the natural expectation of the user is that we can reliably "disable website key bindings". In my experience, that's neither possible nor necessarily desirable. 

Can we pull this out and start a new thread to discuss what you're after here?

Reply:
I tried it and it has no problem like that. I can not take this out of this pr since I developed everything together

Reply:
since it's experimental, I hide .customForce but code in back is still there